### ISSUE TITLE
Is :CALLF [Addr16DW] wrong?

### ISSUE THREAD

**Issue by thomas-haller** [esaulenka/ghidra_STM8#3]
As far as I can read the isa description of the STM8 in the .sinc file the two instructions ":CALLF Addr24" and ":CALLF [Addr16DW]" are defined in a very similar way. But doesn't the ":CALLF [Addr16DW]" need one dereferencing?

I do not have code here which uses this instruction so I cannot check.

**Comment by esaulenka** [comment]
Yes, you are right, this https://github.com/esaulenka/ghidra_STM8/commit/15d9f2e0a9734edc0f52bc2b684fd33f2c38fd68 code looks better.

But I dont have such code too. Needs to write some example to check it. I will try to patch some binary for test.
